README.md

```diff
@@ -422,6 +422,24 @@ done manually by someone on the Azul team. Ask on Slack to find out more.
 
 [Terra Data Repository]: https://jade.datarepo-dev.broadinstitute.org/
 
+### 3.2.2 Personal account registration
+
+In order to register your personal email with SAM the following actions need to
+be performed:
+
+Ensure you are logged in with the correct Google Account:
```

Comment:
Two colons here are very awkward. Based on README precedent, the introductory paragraph should end with a period, and the instruction steps should be numbered.

Reply:
I've omitted the colons for now, perhaps this is something @hannes-ucsc  can clarify with #2169 

Note that we have mixed colon usage for lists of statements for action, some commits to the README add

```
End of action to perform. Run
 Some code block here that helps run that action.

To do some action run:
   Some code block here that helps run that action. 

 To do some action run 
     Some code block here that helps run that action. 

Run 
     Some code block here that helps run that action. 

Then run:
     Some code block here that helps run that action. 
```

The variations of  usage could be due to the age of this README / differences in how we write instructions. Personally I think there should be a guideline to start following so its formatted to look the same.

Interestingly the contributing guidelines use colons before each example code block. While the upgrading.rst has mixed usage.